doc/source/code-examples/advancedexamples.rst

```diff
@@ -78,19 +78,8 @@ or programmatically, during runtime, as follows:
     # retrieve the current number of threads
     current_threads = qibo.get_threads()
 
-On the other hand, when using the ``tensorflow`` backend Qibo inherits
-Tensorflow's defaults for CPU thread configuration.
-Tensorflow allows restricting the number of threads as follows:
-
-.. code-block:: python
-
-    import tensorflow as tf
-    tf.config.threading.set_inter_op_parallelism_threads(1)
-    tf.config.threading.set_intra_op_parallelism_threads(1)
-    import qibo
-
-Note that this should be run during Tensorflow initialization in the beginning
-of the script and before creating the qibo backend.
+For similar wariness when using a machine learning backend (such as TensorFlow or Pytorch)
+please refer to the Qiboml documentation.
```

Comment:
A link here would be probably helpful, even though I don't think we have any right now in qiboml. This is more a reminder to add it, once we have it.

Reply:
I think we can add a simple documentation in Qiboml for now, just with these instructions.